src/pkg/blob/dao/dao.go

```diff
@@ -389,3 +392,19 @@ func (d *dao) DeleteBlob(ctx context.Context, id int64) error {
 	}
 	return nil
 }
+
+func (d *dao) GetBlobsNotRefedByProjectBlob(ctx context.Context, timeWindowHours int64) ([]*models.Blob, error) {
+	var noneRefed []*models.Blob
+	ormer, err := orm.FromContext(ctx)
+	if err != nil {
+		return noneRefed, err
+	}
+
+	sql := fmt.Sprintf(`SELECT b.* FROM blob AS b LEFT JOIN project_blob pb ON b.id = pb.blob_id WHERE pb.id IS NULL AND b.update_time <= now()- interval '%d hours';`, timeWindowHours)
```

Comment:
Is "LEFT JOIN ...... WHERE xx IS NULL" a very common usage?  I would use a `NOT EXIST`...

Reply:
<img width="984" alt="image" src="https://user-images.githubusercontent.com/2841473/86077760-2cb80a00-babf-11ea-9ff9-f0756ec4802e.png">